unpackaged/vesuvio_calibration/calibrate_vesuvio_uranium_martyn_MK5.py

```diff
@@ -379,114 +379,155 @@ def _normalise_sample_by_background(self):
 #----------------------------------------------------------------------------------------
 
   def _fit_bragg_peaks(self):
-    #ESTIMATE PEAK POSITIONS USING D SPACING
-    peak_positions = self._estimate_bragg_peak_positions()
-    num_peaks, num_spectra = peak_positions.shape
+    estimated_peak_positions_all_spec = self._estimate_bragg_peak_positions()
+    num_estimated_peaks, num_spectra = estimated_peak_positions_all_spec.shape
+
     self._prog_reporter = Progress(self, 0.0, 1.0, num_spectra)
 
-    #CREATE OUTPUT PARAMETERS TABLE
-    peaks_table = self._output_workspace_name + '_Peak_Parameters'
-    CreateEmptyTableWorkspace(OutputWorkspace=peaks_table)
+    output_parameters_tbl_name = self._output_workspace_name + '_Peak_Parameters'
+    self._create_output_parameters_table_ws(output_parameters_tbl_name, num_estimated_peaks)
 
-    #PARAM NAMES FOR LINEAR BACKGROUND
-    param_names = ['f0.A0', 'f0.A1']
+    output_workspaces = []
+    for index, estimated_peak_positions in enumerate(estimated_peak_positions_all_spec.transpose()):
+        spec_number = self._spec_list[0]+index
+        self._prog_reporter.report("Fitting to spectrum %d" % spec_number)
```

Comment:
Is it possible to modernise this file to use f-strings?

```suggestion
        self._prog_reporter.report(f"Fitting to spectrum {spec_number}")
```

Reply:
I need to go through the whole script and update a few things, use of `f-strings` is one of them. I will do this in a subsequent PR once this batch has been merged.